src/CorrelationIdService.php

```diff
@@ -39,6 +42,12 @@ public function generateCorrelationId(): string
      */
     public function getCurrentCorrelationId(): ?string
```

Comment:
As this would always return a string, we could update it to:

```suggestion
    public function getCurrentCorrelationId(): string
```

Reply:
Will do it in a later patch, as this would be a backwards compatibility break if someone extends the `CorrelationIdService` as the method signature changes.